xlml/utils/bigquery.py

```diff
@@ -112,6 +112,9 @@ def metadata_history_table_id(self):
 
   def is_valid_metric(self, value: float):
     """Check if float metric is valid for BigQuery table."""
+    if not isinstance(value, float):
```

Comment:
I did not recall if I met any issue for this before. Probably not.

Reply:
The new GKE benchmarks sometimes would have non-float metrics in some of the json output.

Reply:
Are those values expected to be wiped out?

Reply:
They are not used in our metrics, it's ok to discard them.

Reply:
Will integers be filtered out?

```
>>> a = 1
>>> isinstance(1, float)
False
>>> isinstance(a, float)
False
```

Reply:
Can you update it something like 

```
if isinstance(value, numbers.Number) and not (value, bool):
    invalid_values = [math.inf, -math.inf, math.nan]
    return not (value in invalid_values or math.isnan(value))

```